src/main/java/org/gitlab4j/api/GitLabApi.java

```diff
@@ -19,6 +8,12 @@
 import org.gitlab4j.api.utils.Oauth2LoginStreamingOutput;
 import org.gitlab4j.api.utils.SecretString;
 
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.*;
```

Comment:
@devnied Need to specify individual imports, we do not use the wildcard in this library, it can actually cause problems.  This section should simply be reverted to the way it was.

Reply:
Done